src/controllers/UserController.ts

```diff
@@ -0,0 +1,28 @@
+import User, { IUser } from '../models/User';
+import { Request, Response } from "express";
+import asyncHandler from 'express-async-handler';
+
+interface UserController {
+    createUser?: any;
+    getUsers?: any;
+}
+
+const userController: UserController = {};
+
+const createUser = asyncHandler(async (req: Request, res: Response) => {
+    console.log("create users in the controller")
+    console.log(req.body)
+    const newDoc = await User.create(req.body);
```

Comment:
The create Methods is available, because User extends "Document" right?

Reply:
no, because the User is a model -> from this line in the User.ts:
`const User = mongoose.model<IUser>("User", userSchema);`

Reply:
Ok got it, thanks!